packages/@aws-cdk/aws-events-targets/lib/aws-api.ts

```diff
@@ -118,6 +121,18 @@ export class AwsApi implements events.IRuleTarget {
   }
 }
 
+/**
+ * Check if the given service exists in the AWS SDK. If not, a warning will be raised.
+ * @param service Service name
+ */
+function checkServiceExists(service: string, handler: lambda.SingletonFunction) {
+  const sdkService = awsSdkMetadata[service.toLowerCase()];
```

Comment:
Are we sure this is correct for services like `SQS` and such? I'm not too familiar with the contents of this map.

Reply:
This map is generated in buildtools/gen.js. It is built from https://github.com/aws/aws-sdk-js/blob/master/apis/metadata.json

Reply:
Ping @rix0rrr